docs/faq.md

```diff
@@ -14,6 +14,20 @@ curl -fsSL https://ollama.com/install.sh | sh
 
 Review the [Troubleshooting](./troubleshooting.md) docs for more about using logs.
 
+## Is my GPU compatible with Ollama?
+
+### Nvidia
+Ollama supports Nvidia GPUs with compute capability 3.5 to 8.6.
```

Comment:
5.0 is our current minimum.

I have a PR open that would add 3.5, but I don't intend to merge it as is, and at some point I was planning to refactor it to allow building from source to get 3.5+ support but not ship it officially.  In testing, what I see is the 3.5 GPUs are slower than modern CPUs, and to pick them up would either require another variant, or changing compile flags to llama.cpp that may have a negative impact on modern GPUs.